src/reaper_osara.cpp

```diff
@@ -449,12 +449,12 @@ string formatNoteLength(double start, double end) {
 }
 
 string formatLength(double start, double end, TimeFormat timeFormat,
-	FormatTimeCacheRequest cache, bool includeZeros) {
+	FormatTimeCacheRequest cache, bool includeZeros, bool includeProjectStartOffset) {
```

Comment:
Do we have a reasonable degree of certainty that Cockos will change the render behaviour? Do they acknowledge that it's a bug?

Generally, it's best to try to restrict a pull request for a feature to the changes that are necessary for that feature right now. That said, I can live with this if we think it's likely to change in the near term.

Reply:
It'd be good to get this question answered:

> Do we have a reasonable degree of certainty that Cockos will change the render behaviour? Do they acknowledge that it's a bug?

Otherwise, I think we should remove the new formatLength argument from this patch.